src/app/containers/StoryPromo/__snapshots__/index.test.jsx.snap

```diff
@@ -188,39 +194,43 @@ exports[`StoryPromo Container should render correctly for amp 1`] = `
   <div
     className="c1"
   >
-    <div
+    <figure
```

Comment:
Are we sure that inserting a `<figure>` element makes sense in the front page? The issue for this PR reads to me as if the `<figure>` element is optional when using the figure container, but inspecting [the source code](https://github.com/bbc/simorgh/blob/latest/src/app/containers/Figure/index.jsx) shows that that's not the case.

@dr3 Am I reading #1742 correctly?

Reply:
You are reading it correctly @aacn500 figure is not needed. Am making the necessary changes.

Reply:
Updated `FigureContainer` to allow optionally wrapping image in `<figure>`